packages/vc-status-list-issuer/src/agent/StatusListPlugin.ts

```diff
@@ -117,56 +143,129 @@ export class StatusListPlugin implements IAgentPlugin {
     return statusListDetails
   }
 
+  /**
+   * Adds status information to a credential by either:
+   * 1. Using existing status ID from the credential if present
+   * 2. Using provided status list options
+   * 3. Falling back to the default status list ID
+   *
+   * @param args Contains credential and status options
+   * @param context Required agent context
+   * @returns Credential with added status information
+   */
   private async slAddStatusToCredential(args: IAddStatusToCredentialArgs, context: IRequiredContext): Promise<CredentialWithStatusSupport> {
     const { credential, ...rest } = args
+    logger.debug(`Adding status to credential ${credential.id ?? 'without ID'}`)
+
     const credentialStatus = credential.credentialStatus
-    if (!credentialStatus) {
-      logger.info(`Not adding status list info, since no credentialStatus object was present in the credential`)
-      return Promise.resolve(credential)
+    if (credentialStatus) {
+      let existingStatusId: string | undefined
+      if (Array.isArray(credentialStatus)) {
+        // This was implemented with VC v2.0 support, but the rest of the SDK is not ready for that, so ICredential.credentialStatus's array union is disabled for now
```

Comment:
vc 2.0? We do not support that. You mean VCDM

Reply:
Yes I mean this one
https://www.w3.org/TR/vc-data-model-2.0/#example-use-of-multiple-entries-for-the-status-property

Reply:
So vcdm indeed